### PR TITLE
Updated DB volume requirements (bsc#1181369)

### PR DESCRIPTION
**Pull request by tbazant**
closes https://bugzilla.suse.com/show_bug.cgi?id=1181369

### CODE REVIEW

File: xml/deployment_hwrecommend.xml [hunk 1]
```diff
@@ -586,8 +586,10 @@
    <itemizedlist>
     <listitem>
      <para>
-      We recommend reserving 4 GB for the WAL device. The recommended size for
-      DB is 64 GB for most workloads.
+      We recommend reserving 4 GB for the WAL device. While the minimal DB size
+      is 64 GB for RBD-only workloads, the recommended the DB size for &ogw;
```
Comment (ifed01) [inline xml/deployment_hwrecommend.xml:590]: typo? rgw not ogw...

File: xml/deployment_hwrecommend.xml [hunk 2]
```diff
@@ -586,8 +586,10 @@
    <itemizedlist>
     <listitem>
      <para>
-      We recommend reserving 4 GB for the WAL device. The recommended size for
-      DB is 64 GB for most workloads.
+      We recommend reserving 4 GB for the WAL device. While the minimal DB size
```
Comment (lproven) [inline xml/deployment_hwrecommend.xml:589]:
```suggestion
      We recommend reserving 4&nbsp;GB for the WAL device. While the minimal DB size
```

File: xml/deployment_hwrecommend.xml (hunk 1 above)
Comment (lproven) [inline xml/deployment_hwrecommend.xml:590]:
```suggestion
      is 64&nbsp;GB for RBD-only workloads, the recommended DB size for &ogw;
```

File: xml/deployment_hwrecommend.xml [hunk 3]
```diff
@@ -586,8 +586,10 @@
...
+      is 64 GB for RBD-only workloads, the recommended the DB size for &ogw;
+      and &cephfs; workloads is 2% of the main device capacity (but at least
+      196 GB).
```
Comment (lproven) [inline xml/deployment_hwrecommend.xml:592]:
```suggestion
      196&nbsp;GB).
```

**Review by lproven — CHANGES_REQUESTED** [review]

**Pull request merged**